docs/testing/_category_.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
It seems like this would fit better under the `Kuneiform` docs, no? Is there a reason you made it a separate section?

Reply:
Unless the testing also covers oracles and extensions, then maybe a separate section is appropriate.

Reply:
Nah, because it can also test extensions. It cannot test oracles yet (not really sure how that would work), but it could definitely test resolutions if we wanted.

Reply:
Log: Will move into Kuneiform section for now, since all of the features it can test are Kuneiform-related.